test/powershell/engine/Basic/Telemetry.Tests.ps1

```diff
@@ -17,7 +17,6 @@ Describe "Telemetry for shell startup" -Tag CI {
             Rename-Item -Path $uuidPath -NewName "${uuidPath}.original"
         }
 
-        $PWSH = (Get-Process -Id $PID).MainModule.FileName
```

Comment:
If we run tests __locally__ PowerShell do search 'pwsh' by paths from PATH and can run _another_ pwsh version.
I think we should use the pattern everywhere in tests.

Reply:
That's a fair point, but pwsh explicitly puts itself in front of PATH.  So the only edge case is if someone modifies PATH after pwsh startup.

Reply:
Actually, I think changing it to use "$PSHOME/pwsh" makes sense